pkg/goods/support/host-support-bundle.tmpl.yaml

```diff
@@ -448,3 +606,38 @@ spec:
         - pass:
             when: "false"
             message: "No signs of hostname changes found"
+  - textAnalyze:
+      checkName: Check if localhost resolves to 127.0.0.1
+      fileName: host-collectors/run-host/localhost-ips.txt
+      regex: 'localhost has address 127.0.0.1'
+      outcomes:
+        - fail:
+            when: "false"
+            message: "'localhost' does not resolve to 127.0.0.1 ip address"
+        - pass:
+            when: "true"
+            message: "'localhost' resolves to 127.0.0.1 ip address"
+  - textAnalyze:
+      checkName: Check if SELinux is enabled
+      fileName: host-collectors/run-host/sestatus.txt
+      regex: '(?m)^Current mode:\s+enforcing'
+      ignoreIfNoFiles: true
+      outcomes:
+        - fail:
+            when: "true"
+            message: "SELinux is enabled when it should be disabled for kubernetes to work properly"
+        - pass:
+            when: "false"
+            message: "SELinux is disabled as expected"
```

Comment:
Doesn't k0s ship selinux config? Are we sure this one applies to EC?

Reply:
I don't know, but maybe alex can chime in - I would still like to know the current state if it got changed after-the-fact.  thinking along the same lines we should always check ip_forwarding is enabled even if we expect k0s to turn it on

Reply:
Oh nevermind: https://docs.k0sproject.io/v1.31.1+k0s.1/selinux/#set-selinux-labels-for-k0s-installation-files
We don't do those things so it's not configured go ahead and leave this check.

Just need to update the message. "SELinux is not supported, disable it with <insert command here> to proceed."

Reply:
done